RoutesApp_ios/repository/network/managers/FirebaseAuthManager.swift

```diff
@@ -88,6 +88,8 @@ class FirebaseAuthManager: AuthProtocol {
         Auth.auth().settings?.isAppVerificationDisabledForTesting = false
         PhoneAuthProvider.provider().verifyPhoneNumber(phoneNumber, uiDelegate: nil) { (verificationID, error) in
             if let error = error {
+                print("Envio")
+                print(error)
```

Comment:
this prints shouldnt be committed to the repository.
Specifically the print that says Envio

Reply:
sorry , done